azure-kusto-data/tests/inputs/dataframe.json

```diff
@@ -66,55 +66,55 @@
             [
                 "now",
                 "2021-12-22T11:43:00Z",
-                "1.01:01:01.0",
+                "1.00:00:00.0000000",
                 true,
                 5678,
                 3.14159
             ],
             [
                 "earliest datetime",
                 "1677-09-21T00:12:44Z",
```

Comment:
should we add an earlier date here, to see if we still panic w/o error?

Reply:
Done, we get None.